content/news/2023-10-13-EGD-2023-the-wrapup/index.md

```diff
@@ -25,38 +25,36 @@ subsites: [all, esg]
 
 </div>
 
-About a week ago the [2023 edition of the European Galaxy Days (EGD 2023)](/events/2023-10-egd) has ended, and we can say: it was a pleasure to have you as guests in Freiburg, including those who participated virtually. The three days were as intense as exciting, no matter whether during the talks, between the session at coffee or lunch, or in the (partly long) evenings in various gastronomical outposts in the city.
+About a week ago the [2023 edition of the European Galaxy Days (EGD 2023)](/events/2023-10-egd) has ended, and we can say: it was a pleasure! The three days were as intense as exciting, no matter whether during the talks, between the session at coffee or lunch, or in the (partly long) evenings in various gastronomical outposts in the city.
  
-Not least, the EGD were a great success, especially in the light of the fact that both planning and advertising for the event have been initiated on a quite short notice. While for many of us this year's [GCC in Australia](/events/gcc2023/) appeared even less affordable than the [US iteration in 2022](/events/gcc2022/), we felt on duty to offer again some time and space to you in the middle of Europe to gather together. 
-
-Like for the [EGD 2022](/events/2022-10-egd/), effectively we had a triple event also in 2023, joining forces again with [ELIXIR](https://elixir-europe.org) and [EuroScienceGateway](/projects/esg/) (ESG), a project from the [EOSC space](https://eosc.eu), which effectivly is driven by the European Galaxy community. 
+Not least, the EGD were a great success. While for many of us this year's [GCC in Australia](/events/gcc2023/) was too far away, we loved to offer again some time and space to you in the middle of Europe to gather together. Close to 80 participants joined, and like for the [EGD 2022](/events/2022-10-egd/), effectively we had a triple event this year, too. We joined forces again with [ELIXIR's](https://elixir-europe.org) Galaxy community, as well as [EuroScienceGateway](/projects/esg/) (ESG), a project from the [EOSC space](https://eosc.eu), which effectivly is driven by the European Galaxy community. 
```

Comment:
two times effectivly
I would remove the last part of the sentence. There is not much information in it and it could exclude people. Especially our new communities and EGI.